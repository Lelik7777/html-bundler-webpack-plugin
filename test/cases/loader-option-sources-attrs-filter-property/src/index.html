<html>
  <head>
    <title>Test</title>
    <meta property="og:title" content="Frutis" />
    <meta property="og:image" content="./frutis.png" />
    <meta property="og:image:type" content="image/png" />
    <meta property="og:video" content="./video.mp4" />
    <meta property="og:video:type" content="video/mp4" />
  </head>
  <body>
    <h1>Home</h1>
    <img src="./apple.png">
  </body>
</html>