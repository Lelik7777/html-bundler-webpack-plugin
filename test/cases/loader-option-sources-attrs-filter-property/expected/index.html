<html>
  <head>
    <title>Test</title>
    <meta property="og:title" content="Frutis" />
    <meta property="og:image" content="assets/img/frutis.697ef306.png" />
    <meta property="og:image:type" content="image/png" />
    <meta property="og:video" content="assets/video/video.697ef306.mp4" />
    <meta property="og:video:type" content="video/mp4" />
  </head>
  <body>
    <h1>Home</h1>
    <img src="assets/img/apple.697ef306.png">
  </body>
</html>